sdk/confidentialledger/tests.yml

```diff
@@ -0,0 +1,16 @@
+trigger: none
+
+stages:
+  - template: ../../eng/pipelines/templates/stages/archetype-sdk-tests.yml
+    parameters:
+      AllocateResourceGroup: false
```

Comment:
It seems there are no resource deployments required for these live tests. @seankane-msft is this field the appropriate way to skip, or is this legacy?

Reply:
Yes that is correct, the default is True